Intersections of Two Arrays II.py

```diff
@@ -0,0 +1,33 @@
+# TC : O(NlogN) - N is the length of the longest array
+# SC : O(1)
```

Comment:
SC is O(n) due to sorting

Reply:
Please mention if it ran successfully on leetcode or not, for every problem
- Sure
SC is O(n) due to sorting
- Sorting happens in place. Right?

Reply:
To get TC as O( n log n) for sorting you have to use either merge sort or quick sort and for both the algorithms SC is not O(1), it would be in between O(log n) to O(n)

Reply:
Ohh got it, I thought Tim sort happens in place.